scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java

```diff
@@ -170,6 +173,7 @@ public SchedulerPage(SchedulerController controller) {
         this.controller.getEventDispatcher().addSchedulerStatusListener(this);
         LogModel.getInstance().addLogListener(this);
         this.controller.getExecutionController().getModel().addExecutionsDisplayModeListener(this);
+        // very ugly, only way to control the scroll viewport in VisualizationViewImage.java
         inst = this;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this assignment of "inst". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3010)